src/main/query_profiler.cpp

```diff
@@ -239,6 +242,9 @@ void QueryProfiler::EndQuery() {
 
 string QueryProfiler::ToString(ExplainFormat explain_format) const {
 	const auto format = GetPrintFormat(explain_format);
+	if (!IsEnabled()) {
```

Comment:
@Tishj when will `IsEnabled()` return false? It looks like this is always going to return true in the case of `explain analyze`?